app/Filament/Technicien/Pages/FillInterventionReportPage.php

```
@@ -0,0 +1,179 @@
+
+    public ?InterventionReportTemplate $template = null;
+
+    public function mount(?int $intervention_id = null): void
+    {
+        $this->intervention_id = $intervention_id ?: (int) request()->query('intervention_id');
+
+        if (! $this->intervention_id) {
+            abort(404);
+        }
+
+        $this->intervention = Intervention::findOrFail($this->intervention_id);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Interventions\Intervention::findOrFail().


(staticMethod.notFound)